### ISSUE TITLE
Make offline builds (relatively) easy

### ISSUE THREAD

**Issue by auscompgeek** [robotpy/robotpy-build#42]
Imagine a build service that doesn't allow network access.

This would make it easy for me to build packages for various distros.

**Comment by virtuald** [comment]
You can pass the location of the cache directory to build_dl

**Comment by auscompgeek** [comment]
Right, sure. But I'd also have to figure out what to download beforehand.

**Comment by virtuald** [comment]
build_dl will do that.
